Big-Spender/readme.md

```diff
@@ -113,6 +119,7 @@ INSERT YOUR QUERY HERE
 
 ```sql
 CREATE YOUR QUERY HERE
+SELECT SUM(amount), date FROM spends GROUP BY date;
```

Comment:
okay-- this gives us the two dates, BUT what if we had other dates? like 2021-03-20 ? What would happen with the data here?